ui/pages/extension/AccessMesheryModal.js

```diff
@@ -0,0 +1,106 @@
+import React, {
+} from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { Typography, Grid, Button, IconButton } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+
+const styles = makeStyles((theme) => ({
+  paper : {
+    position : 'absolute',
+    width : "30rem",
+    height : "20rem",
```

Comment:
move away from static sizing where you can. use max-height, please with overflow set to scroll.

<img width="587" alt="Screenshot 2022-12-16 at 4 28 13 PM" src="https://user-images.githubusercontent.com/7570704/208199434-a6b8f9eb-6a12-4002-9bbb-ce783ea5a6ad.png">

Reply:
Done